internal/scaffold/cmd.go

```diff
@@ -162,15 +163,15 @@ func main() {
 	}
 	// Create Service object to expose the metrics port(s).
 	service, err := metrics.CreateMetricsService(ctx, cfg, servicePorts)
-	if err != nil {
+	if err != nil && !errors.Is(err, k8sutil.ErrRunLocal) {
```

Comment:
Looking at the source for `CreateMetricsService`, `ErrRunLocal` is checked but [not returned](https://github.com/operator-framework/operator-sdk/blob/master/pkg/metrics/metrics.go#L56).